tests/Feature/WrongMemberControllerTest.php

```diff
@@ -0,0 +1,369 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Model\MemberModel;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class WrongMemberControllerTest extends TestCase
+{
+    //Wrong testing cases:
+    use DatabaseMigrations;
+
+    //test add.name 101 charater
+    public function testAddName101Charaters()
+    {
+        $newMember = [
+            'name' => '01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567891',
+            'address' => 'NamTuLiem,HaNoi',
+            'age' => 23,
+            'email' => 'romecody@gmail.com',
+        ];
+
+        $response = $this->call('POST', route('post.add'), $newMember);
+        $this->assertEquals(302, $response->status());
```

Comment:
If the controller also gives a JSON response, please assert the response message.